dis_snek/models/discord/user.py

```diff
@@ -115,6 +116,14 @@ def _process_dict(cls, data: Dict[str, Any], client: "Snake") -> Dict[str, Any]:
 
         return data
 
+    @property
+    def member_instances(self) -> List["Member"]:
+        """Returns the member object for all guilds both the bot and the user are in. Note: This will only be accurate if the guild members are cached internally"""
```

Comment:
Notes format seems to be inaccurate

Reply:
Specifically `!!!` is for notes `???` is for collapsable notes